options/locale/locale_en-US.ini

```diff
@@ -149,7 +149,7 @@ lfs_path = Git LFS Root Path
 lfs_path_helper = Files tracked by Git LFS will be stored in this directory. Leave empty to disable.
 run_user = Run As Username
 run_user_helper = Enter the operating system username that Gitea runs as. Note that this user must have access to the repository root path.
-domain = SSH Server Domain
+domain = Server Domain
 domain_helper = Domain or host address for SSH clone URLs.
```

Comment:
Please update this too.

Reply:
can't see it's referenced: https://github.com/go-gitea/gitea/search?p=1&q=domain_helper

maybe remove the helper?

Reply:
https://github.com/go-gitea/gitea/blob/main/templates/install.tmpl#L119

Reply:
Otherwise LGTM

Reply:
updated others also